requirements.txt

```diff
@@ -0,0 +1,77 @@
+aiocsv==1.3.2
+aiofile==3.8.8
+aiofiles==24.1.0
+aiohappyeyeballs==2.4.3
+aiohttp==3.10.8
+aiomultiprocess==0.9.1
+aiopath==0.7.7
+aioshutil==1.5
+aiosignal==1.3.1
+aiosmtplib==3.0.2
+annotated-types==0.7.0
+anyio==4.6.0
+art==6.3
+async-unzip==0.3.6
+asyncio-pool==0.6.0
+attrs==24.2.0
+caio==0.9.17
+certifi==2024.8.30
+charset-normalizer==3.3.2
+click==8.1.7
+colorama==0.4.6
+coloredlogs==15.0.1
+dotwiz==0.4.0
+elastic-transport==8.15.0
+elasticsearch==8.15.1
+Events==0.5
+evtx==0.8.0
+fastapi==0.115.0
+frozenlist==1.4.1
+greenlet==3.1.1
+git+https://github.com/mentat-is/gulp.git@f886d030e6b447e66f46c73916d503e8b1340757#egg=gulp
```

Comment:
i'm not a python expert, tbh i'm not sure if this is a best practice or not. same for the dependency below

Reply:
fair enough, for the future this looks like what you need https://docs.github.com/en/actions/use-cases-and-examples/building-and-testing/building-and-testing-python#publishing-to-pypi